### PR TITLE
docs: PEM-1296 removed web shell language

### PR DESCRIPTION
**Pull request by karl-cardenas-coding**
## Describe the Change

This PR removes the web shell language until further notice.

## Review Changes

💻 [Preview](https://deploy-preview-1182--docs-spectrocloud.netlify.app/clusters/cluster-management/palette-webctl)

🎫 [Jira Ticket](https://spectrocloud.atlassian.net/browse/PEM-1296)

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1182#" title="Author self-approved">karl-cardenas-coding</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: content/docs/04-clusters/06-cluster-management/12-palette-webctl.md
Comment (ritawatson) [inline content/docs/04-clusters/06-cluster-management/12-palette-webctl.md:75]:
```suggestion
Verify you have access to your host cluster by issuing kubectl commands against it. 
```

**Review by ritawatson — APPROVED** [review]
LGTM - just one wording suggestion for the Validation section. It's up to you.

**Pull request merged**
